From 1b051e80eb61bd48aae636fddf119a93828e7caa Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Wed, 3 Sep 2025 19:27:54 +0200
Subject: [PATCH] Add trailing semicolon

---
 asyncpg/protocol/record/recordobj.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/asyncpg/protocol/record/recordobj.c b/asyncpg/protocol/record/recordobj.c
index 9767f43..d983994 100644
--- a/asyncpg/protocol/record/recordobj.c
+++ b/asyncpg/protocol/record/recordobj.c
@@ -117,7 +117,7 @@ record_dealloc(ApgRecordObject *o)
     }
     Py_TYPE(o)->tp_free((PyObject *)o);
 done:
-    Py_TRASHCAN_END
+    Py_TRASHCAN_END;
 }
 
 
