From a68e15184c85797202d3951d49956f6ee48abc4a Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Mon, 13 Nov 2023 09:59:32 -0500
Subject: [PATCH] Allow Cython 3 (fix #1083)

---
 pyproject.toml | 2 +-
 setup.py       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 12f6ae68..2da1b880 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -59,7 +59,7 @@ requires = [
     "setuptools>=60",
     "wheel",
 
-    "Cython(>=0.29.24,<3.0.0)"
+    "Cython(>=0.29.24,<4.0.0)"
 ]
 build-backend = "setuptools.build_meta"
 
diff --git a/setup.py b/setup.py
index f7c3c471..5f2709d7 100644
--- a/setup.py
+++ b/setup.py
@@ -25,7 +25,7 @@
 from setuptools.command import build_ext as setuptools_build_ext
 
 
-CYTHON_DEPENDENCY = 'Cython(>=0.29.24,<0.30.0)'
+CYTHON_DEPENDENCY = 'Cython(>=0.29.24,<4.0.0)'
 
 CFLAGS = ['-O2']
 LDFLAGS = []
